pkgs/applications/video/mplayer/default.nix

```diff
@@ -199,6 +199,6 @@ stdenv.mkDerivation rec {
     homepage = "http://mplayerhq.hu";
     license = "GPL";
     maintainers = [ stdenv.lib.maintainers.eelco stdenv.lib.maintainers.urkud ];
-    platforms = stdenv.lib.platforms.linux;
+    platforms = stdenv.lib.platforms.unix;
```

Comment:
I personally don't like being to restrictive with the platforms, IMHO this should include all the platforms that are supported upstream even if tjey doesn't necessarily work (yet) in nixpkgs.